DuckDuckGo/NativeDisplayableCertificateBuilderDriver.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  NativeDisplayableCertificateBuilderDriver.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+class NativeDisplayableCertificateBuilderDriver: DisplayableCertificateBuilderDriver {
+
+    func build(usingTrust trust: SecTrust) -> DisplayableCertificate {
+
+        var head: DisplayableCertificate!
+        var next: DisplayableCertificate!
```

Comment:
Do you have to force unwrap head and next?

Reply:
Yep, otherwise the compiler says they may not have been initialised.

Reply:
Will improve the 0 certs edge case though by returning the error constant.